Update offboarding page (#98)
cct-datascience/group-procedures: offboarding.qmd

With more time, the group can help with next steps including finding new positions and creating an exit plan.

> Further, it typically takes a month or more to effectively document and transfer knowledge of what you are working on.

Reviewer: I know this section wasn't edited in this PR, but I gotta say I don't *love* the content of this paragraph.  There are plenty of reasons someone might not want to tell their boss that they are looking for another job and there's definitely no actual requirement to disclose this, so I think I'd feel more comfortable if the wording of this was "softened" a bit.  Like, ideally everyone is comfortable discussing career moves with Kristina and sure it helps to get advanced notice, but we have processes in place to ensure that two weeks notice hopefully *is* plenty of time to transfer projects. Everyone ideally should be documenting projects as they go, not spending the month before they move on doing all the documentation on all their projects.  We have processes in place so everyone is more or less up to date on the status of all the projects we are working on, so offboarding shouldn't take a month or more.

### Before final day

In approximately chronological order

-   Inform Kristina about what your last day will be
-   Review University of Arizona's [official "leaving" page](https://hr.arizona.edu/employees-affiliates/leaving-ua)
-   Write resignation letter for UA
    -   Kristina will send this and final date to HR & ALVSCE business office

> -   Review two university checklists in Google Drive [offboarding folder](https://drive.google.com/drive/folders/1t__w3Lgrf7uju6SxXb0pGkfNE0LlTdFQ?usp=drive_link) and complete relevant tasks

Reviewer: Do we want to be responsible for keeping these up-to-date, or should we instead link to these on the HR page (if possible)?

Author: Because these checklists probably won't change substantially over time, while links to the HR website pages probably will, I think I will leave it this way for the time being.

-   Decide if you should become a DCC
    -   [UA DCC guidelines](https://hr.arizona.edu/employees/dccs)
    -   This might make sense if you are continuing collaborations with UA folks, or need to retain access to UA services such as email or Google Drive
    -   This process is initiated before leaving and can only be finalized after final day

> -   Start to transfer relevant knowledge about projects and trainings
>     -   This process could take a substantial amount of time
>     -   Who should know what depends on the circumstance

Reviewer: ```suggestion
-   Start to transfer relevant knowledge about projects and trainings
    -   This process could take a substantial amount of time
    -   Focus on bringing documentation up to date
    -   Meet with relevant people to transfer knowledge and train them as needed
```

Just an idea to make this a little more "actionable"

-   Transition resources on digital accounts
    -   **Google Drive**
        -   Move all documents that could be useful for the group from your personal UA Google Drive to the UA CCT DS Google Drive
        -   Any docs you created in the UA CCT DS Google Drive should be accessible to rest of group; we all have default editor access
    -   **GitHub**

>         -   Make a list of all repos you have contributed to for the group, in all organizations and in personal acccount

Reviewer: Any repos that are group-related should probably not be in personal GitHub account to begin with, but if there are any, they can be transferred to `cct-datascience` at this point.

Author: ```suggestion
        -   Make a list of all repos you have contributed to for the group, in all organizations; if there are any repos in personal account, transfer them to our `cct-datascience` organization or another relevant collaborator
```

Author: Good call, I added this clarification

>     -   **Posit Connect**
>         -   For anything deployed to Connect, either add other team members as collaborators or document where the source document is so that others can deploy it later

Reviewer: ```suggestion
    -   **Posit Connect**
        -   For anything deployed to Connect, either add other team members as collaborators or document where the source document is (e.g. by adding a link to a GitHub repo in a Shiny app) so that others can deploy it later
```

>     -   **HPC**
>         -   Make sure all files are in the group folders, as you will lose access on last day

Reviewer: ```suggestion
    -   **HPC**
        -   Make sure all files are in the group folders, as you may lose access after your last day
```

From the [HPC FAQ](https://uarizona.atlassian.net/wiki/spaces/UAHPC/pages/75993121/FAQ) it sounds like you might be able to maintain access if you become a DCC.  This is still a good idea though.

Author: This looks good, and I'll add HPC access to the DCC decision comments earlier on the page

    -   **toggl**
        -   Ensure time tracking is up to date as possible before leaving so we can have accurate project hours estimates

> -   Leave contact information (e.g., personal email address) in case we need to reach you

Reviewer: Should "schedule an exit interview" be on the list somewhere?  If this is not happening, then the Exit Interview section should be updated/removed.

Author: Oh, good call!
